vendor/github.com/emicklei/go-restful/v3/response.go

```diff
@@ -14,7 +14,7 @@ import (
 // DefaultResponseMimeType is DEPRECATED, use DefaultResponseContentType(mime)
 var DefaultResponseMimeType string
 
-// PrettyPrintResponses controls the indentation feature of XML and JSON serialization
+//PrettyPrintResponses controls the indentation feature of XML and JSON serialization
```

Comment:
So the vendor does not have the white space, and every time I saved it would add it. Which would break the module verification because something is different between the way I saved and and the way it gets pulled. We shouldn't affect the vendor directory at all IMO, so I kept it as we get it from a go mod vendor.

Reply:
Gotcha, looks good to me then. However, what do you mean by 
> k8s and runtime-controller due to issues that need to be resolved

Reply:
So there is a bug in the controller-runtime library from version 0.15 and newer that is not fixed yet https://github.com/kubernetes-sigs/controller-runtime/pull/2425 is the fix, but it is not merged. So until that is merged we cannot upgrade. For kubernetes a bunch of function arguments changed and some behavior that I didn't want to mess with in this PR.

Reply:
for future ref this is how this (and similar other issues) are overcome in 0.15 https://github.com/kubevirt/hyperconverged-cluster-operator/pull/2333/files#r1200311040